### PR TITLE
Bazzarvoice documentation

### PR DESCRIPTION
**Pull request by lenadoc**
## PR Description

https://spryker.atlassian.net/browse/APPS-2521

File for review: https://github.com/spryker/spryker-docs/pull/648/files#diff-b9f05f524910874fd57f7a98e34c96b055430ea49c2257479459f160e2218891

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/aop/user/apps/bazzarvoice.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: BazzarVoice
```
Comment (marvkrug) [inline docs/aop/user/apps/bazzarvoice.md:2]: BazaarVoice

File: docs/aop/user/apps/bazzarvoice.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+description: Find out how you can integrate and use BazzarVoice in your Spryker shop
```
Comment (marvkrug) [inline docs/aop/user/apps/bazzarvoice.md:3]: BazaarVoice

File: docs/aop/user/apps/bazzarvoice.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+template: howto-guide-template
+---
+
+The [BazzarVoice](https://www.bazaarvoice.com/) app lets you collect and add user-generated content (UGC) to your product pages.
```
Comment (marvkrug) [inline docs/aop/user/apps/bazzarvoice.md:7]: BazaarVoice

File: docs/aop/user/apps/bazzarvoice.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+1. In the top right corner of the BazzarVoice app details page, click **Connect app**.
+   This takes you to the BazzarVoice site with the signup form.
+2. Fill out the BazzarVoice signup form and submit it.
+   You should receive the BazzarVoice credentials.
+3. Go back to your store's Back Office, to the BazzarVoice app details page.
+4. In the top right corner of the BazzarVoice app details page, click **Configure**.
+5. In the *Configure* pane, enter the credentials you received from BazzarVoice.
+
+That's it. You have integrated the BazzarVoice app into your store. It usually takes BazzarVoice a few days to process your product feed. Therefore, you should see the external ratings and reviews from BazzarVoice in about 2-3 days after you integrated the app.
```
Comment (marvkrug) [inline docs/aop/user/apps/bazzarvoice.md:51]: (There will be some new content for this line, as we are changing from a product feed approach to a dynamic catalog collection approach. Will give you new info ASAP.)

**Review by marvkrug — COMMENTED** [review]
Make sure to use the spelling: BazaarVoice

New content is coming for the product feed paragraph

**Review by VadymSachenko — APPROVED** [review]
Approved from my side.
Please take a look at the comments.

**Pull request merged**